### PR TITLE
Make NEP6Wallet.path public

### PR DESCRIPTION
**Pull request by belane**

### CODE REVIEW

File: src/neo/Wallets/NEP6/NEP6Wallet.cs [hunk 1]
```diff
@@ -265,6 +265,22 @@ public void Save()
             File.WriteAllText(path, wallet.ToString());
         }
 
+        public bool Backup()
```
Comment (erikzhang) [inline src/neo/Wallets/NEP6/NEP6Wallet.cs:268]: Can this method be moved to `Wallet.cs`?

Comment (erikzhang) [reply src/neo/Wallets/NEP6/NEP6Wallet.cs:268]: Or I think this is simply copying the file, there is no need to create a new method. The client can directly call `File.Copy()` to back up any file.

Comment (belane) [reply src/neo/Wallets/NEP6/NEP6Wallet.cs]: ok, then we need `path` public.

File: src/neo/Wallets/NEP6/NEP6Wallet.cs [hunk 2]
```diff
@@ -13,13 +13,13 @@ namespace Neo.Wallets.NEP6
...
     public class NEP6Wallet : Wallet
     {
-        private readonly string path;
         private string password;
         private string name;
         private Version version;
         private readonly Dictionary<UInt160, NEP6Account> accounts;
         private readonly JObject extra;
 
+        public readonly string path;
```
Comment (shargon) [inline src/neo/Wallets/NEP6/NEP6Wallet.cs:22]: Uppercase please :)

**Comment by shargon** [comment]
I think that we should have the property `Path` (as abstract), in `Wallet.cs`

**Pull request merged**